packages/expo-ui/ios/SingleChoiceSegmentedControlView.swift

```diff
@@ -0,0 +1,29 @@
+// Copyright 2025-present 650 Industries. All rights reserved.
+
+import SwiftUI
+import ExpoModulesCore
+
+struct SingleChoiceSegmentedControlView: ExpoSwiftUI.View {
+  @State var selection: Int?
+  @EnvironmentObject var props: SingleChoiceSegmentedControlProps
+  var body: some View {
+    Picker("", selection: $selection) {
```

Comment:
It's okay for now, but we need to pass the title somehow rather than an empty string.